pkg/ir/xds.go

```diff
@@ -0,0 +1,74 @@
+package ir
+
+// Xds holds the intermediate representation of a Gateway and is
+// used by the xDS Translator to convert it into xDS resources.
+type Xds struct {
+	// Name of the Xds IR.
+	Name string
+	// HTTP listeners exposed by the gateway.
+	HTTP []HTTPListener
+}
+
+// HTTPListener holds the listener configuration.
+type HTTPListener struct {
+	// Name of the HttpListener
```

Comment:
nit: s/Http/HTTP/

Reply:
fixed in https://github.com/envoyproxy/gateway/pull/125